<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CEnterWindow</class>
 <widget class="QDialog" name="CEnterWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>429</width>
    <height>235</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Network connection</string>
  </property>
  <widget class="QLineEdit" name="m_ip">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#m_ip {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
padding: 8px;
}</string>
   </property>
   <property name="text">
    <string>172.26.66.243</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>35</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color : white;
}</string>
   </property>
   <property name="text">
    <string>Ip Address</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_port">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#m_port {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
padding: 8px;
}</string>
   </property>
   <property name="text">
    <string>5678</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>44</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color : white;
}
</string>
   </property>
   <property name="text">
    <string>Port</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#connectButton {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
}
</string>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="NameLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>106</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
color : white;
}</string>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="m_name">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Long Shot Cyrillic</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">#m_name {
background-color: transparent;
border-image: url(:/GUI/btn2.jpg);
background: none;
border: none;
background-repeat:none;
color: white;
padding: 8px;
}</string>
   </property>
   <property name="text">
    <string>user</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
